frontend/src/model.tsx

```diff
@@ -30,7 +30,7 @@ export interface TimeEntry {
   id: number;
   issue_id: number;
   activity_id: number;
-  hours: number | string;
+  hours: number;
```

Comment:
This (and everything related to it in other parts of the code) was needed to make it possible to have decimal numbers in cells.

Reply:
I missed this. And indeed without it, you are not allowed to fill in the fractional part.

Reply:
I added a commit fixing the decimal numbers. Sorry I was not aware we wanted to allow decimals. Thanks for the input. I would rather keep the rest of the changes in this PR. I still think we should avoid having two possible types in the TimeEntry. I can add more documentation to the PR later, I need to take Amelia to school now. We can also the PR together later if you want.